docs/getting-started/aspnet5/existing-db.rst

```diff
@@ -0,0 +1,207 @@
+
+In this walkthrough, you will build an ASP.NET 5 MVC application that performs basic data access using Entity Framework.  You will use reverse engineering to create an Entity Framework model based on an existing database.
+
+.. contents:: `In this article:`
+    :depth: 2
+    :local:
+
+.. include:: /_shared/sample.txt
+.. _sample: https://github.com/aspnet/EntityFramework.Docs/tree/master/docs/getting-started/aspnet5/sample
+
+Prerequisites
+-------------
+
+The following prerequisites are needed to complete this walkthrough:
+
+  * Visual Studio 2015
+  * `ASP.NET 5 RC1 Tools for Visual Studio <https://docs.asp.net/en/latest/getting-started/installing-on-windows.html>`_
+  * `Blogging database`_
+
+Blogging database
+^^^^^^^^^^^^^^^^^
+
+This tutorial uses a **Blogging** database on your LocalDb instance as the existing database.
+
+.. include:: /getting-started/_shared/create-blogging-database-vs.txt
+
+Create a new project
+--------------------
+
+	* Open Visual Studio 2015
+	* :menuselection:`File --> New --> Project...`
+	* From the left menu select :menuselection:`Templates --> Visual C# --> Web`
+	* Select the **ASP.NET Web Application** project template
+	* Ensure you are targeting .NET 4.5.1 or later
+	* Enter **EFGetStarted.AspNet5.ExistingDb** as the name and click **OK**
+	* Wait for the **New ASP.NET Project** dialog to appear
+	* Under **ASP.NET 5 Preview Templates** select **Web Application**
+	* Ensure that **Authentication** is set to **No Authentication**
+	* Click **OK**
+
+Install Entity Framework
+----------------------------------------
+To use EF7 you install the package for the database provider(s) you want to target. This walkthrough uses SQL Server. For a list of available providers see :doc:`/providers/index`.
+
+ 	* :menuselection:`Tools --> NuGet Package Manager --> Package Manager Console`
+	* Run ``Install-Package EntityFramework.MicrosoftSqlServer –Pre``
+
+.. note::
+    In ASP.NET 5 projects the ``Install-Package`` will complete quickly and the package installation will occur in the background. You will see **(Restoring...)** appear next to **References** in **Solution Explorer** while the install occurs.
+
+To enable reverse engineering from an existing database we need to install a couple of other packages too.
+
+  * Run ``Install-Package EntityFramework.Commands –Pre``
+  * Run ``Install-Package EntityFramework.MicrosoftSqlServer.Design –Pre``
+  * Open **project.json**
+  * Locate the ``commands`` section and add the ``ef`` command as shown below
+
+.. literalinclude:: sample/src/EFGetStarted.AspNet5.ExistingDb/project.json
+        :language: json
+        :linenos:
+        :lines: 26-29
+        :emphasize-lines: 3
+
+Reverse engineer your model
+---------------------------
+
+.. caution::
+    The reverse engineer experience in ASP.NET 5 is still a work-in-progress. The following steps are overly complex and will be simplified by the time we reach a stable release.
```

Comment:
The main thing is enabling Init.ps1 in xproj, so that we can use PowerShell commands... then it just becomes the same as the other project types (i.e. no dropping out to cmd)